frontend/src/components/movie/MovieView.js

```diff
@@ -57,33 +37,48 @@ class MovieView extends Component {
     };
 
     async loadMovieData(movieId) {
-        const movieDataServer = await this.load(`/api/movie/${movieId}`);
+        const movieDataServer = await get(`/api/movie/${movieId}`);
```

Comment:
the implementation of "get" don't perform this:
 if (!resp.ok) {
            throw Error(resp.statusText);
maybe we should axios?
it means the error that we will see in browser/log will not be 404/500 but the error trying to get the json. something like "unexpected token"